agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/perfcounter/PerformanceCounterContainer.java

```diff
@@ -146,7 +149,7 @@ public void run() {
               } catch (ThreadDeath td) {
                 throw td;
               } catch (Throwable t) {
-                try {
+                try (MDC.MDCCloseable ignored = CUSTOM_JMX_METRIC_ERROR.makeActive()) {
```

Comment:
i can rename it to performance_counter_error? apply to all?

Reply:
I like the custom jmx metric error being separate, since that's related to customer config, compared to "standard" cpu/mem metrics which we provide

though I'm not sure the above ever throws exception here since we generally catch exception downstream before returning up to this point